docs/development_suite/deploy/overview.md

```diff
@@ -160,6 +160,17 @@ If you are not following sematic versioning, for example using `image:latest` or
 It is not recommended to activate the checkbox in a production runtime environment, because if some of your services by mistake have a pre-release or latest version tag they will be redeployed.
 :::
 
+
+### Export and review configuration files
+
+When using **Enhanced Project Workflow**, Kubernetes files are generated on deployment. After selecting the desired revision (or version), you will have the possibility to export and download the files in `tag.gz` format, or to review them directly inside the browser.
```

Comment:
I would put this "enhanced workflow only" information even more prominently with a :::info or with a :::note, and with a link to the enhanced workflow page